x-pack/plugins/observability/public/pages/alerts/components/alert_actions.tsx

```diff
@@ -190,6 +201,19 @@ export function AlertActions({
         </EuiContextMenuItem>
       ),
     ],
+    ...(ruleSavePermissions
```

Comment:
@maryam-saeidi How's tying this to the `update` case permission sound? Or do we need to create an entirely new permission for this?

Reply:
Talked to @XavierM, turns out we don't need permissions on the client side because the user can only see alerts from rules that they're authorized to interact with. I need to implement some auth checking on the server side though.

Reply:
Thanks for the update 👍🏻